### PR TITLE
[zh-cn] Rresync manual rotation of CA

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/tasks/tls/manual-rotation-of-ca-certificates.md [hunk 1]
```diff
@@ -277,21 +317,31 @@ Configurations with a single API server will experience unavailability while the
...
 
-    1. 重启 kubelet，移除 `clientCAFile` 标志所指向的文件以及 kubelet kubeconfig 文件中
-       的老的 CA 数据。
+    1. 在每个节点上，移除 `clientCAFile` 标志所指向的文件，以删除老的 CA 数据，并从
+       kubelet kubeconfig 文件中去掉老的 CA，重启 kubelet。
+       你应该用滚动更新的方式来执行这一步骤的操作。
+
+       如果你的集群允许你执行这一变更，你也可以通过替换节点而不是重新配置节点的方式来将其上线。
+
+
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/tls/manual-rotation-of-ca-certificates.md:347]: The redundant newlines could be removed.

**Review by Sea-n — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/tasks/tls/manual-rotation-of-ca-certificates.md [hunk 2]
```diff
@@ -121,21 +133,25 @@ Configurations with a single API server will experience unavailability while the
...
 
-1. Append the both old and new CA to the file against `-client-ca-file` and `-kubelet-certificate-authority` flag in the `kube-apiserver` configuration.
+1. Append the both old and new CA to the file against `-client-ca-file` and `-kubelet-certificate-authority`
+   flag in the `kube-apiserver` configuration.
 
 1. Append the both old and new CA to the file against `-client-ca-file` flag in the `kube-scheduler` configuration.
 -->
-4. 重启所有使用集群内配置的 Pods（例如：`kube-proxy`、`coredns` 等），以便这些 Pod 能够使用
-   来自 *ServiceAccount* Secret 中的、已更新的证书机构数据。
+4. 重启所有使用集群内配置的 Pod（例如：`kube-proxy`、CoreDNS 等），以便这些 Pod
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/tls/manual-rotation-of-ca-certificates.md:148]: I prefer to not quote single one, looks inconsistent
```suggestion
4. 重启所有使用集群内配置的 Pod（例如：kube-proxy、CoreDNS 等），以便这些 Pod
```

File: content/zh-cn/docs/tasks/tls/manual-rotation-of-ca-certificates.md [hunk 3]
```diff
@@ -121,21 +133,25 @@ Configurations with a single API server will experience unavailability while the
...
+   能够使用与 *ServiceAccount* 相关联的 Secret 中的、已更新的证书机构数据。
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/tls/manual-rotation-of-ca-certificates.md:149]:
```suggestion
   能够使用与 ServiceAccount 相关联的 Secret 中的、已更新的证书机构数据。
```

File: content/zh-cn/docs/tasks/tls/manual-rotation-of-ca-certificates.md [hunk 4]
```diff
@@ -234,41 +268,47 @@ Configurations with a single API server will experience unavailability while the
...
    1. Validate logs from any aggregated api servers and pods using in-cluster config.
 -->
-10. 验证集群的功能正常
+11. 验证集群的功能正常。
 
-    1. 验证控制面组件的日志，以及 `kubelet` 和 `kube-proxy` 的日志，确保其中没有
-       抛出 TLS 错误，参阅
-       [查看日志](/zh-cn/docs/tasks/debug/debug-cluster/#looking-at-logs).
+    1. 检查控制面组件以及 `kubelet` 和 `kube-proxy` 的日志，确保其中没有抛出 TLS 错误，
+       参阅[查看日志](/zh-cn/docs/tasks/debug/debug-cluster/#looking-at-logs).
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/tls/manual-rotation-of-ca-certificates.md:311]:
```suggestion
       参阅[查看日志](/zh-cn/docs/tasks/debug/debug-cluster/#looking-at-logs)。
```

**Pull request merged**